node/net.ts

```diff
@@ -1807,6 +1807,9 @@ function _emitErrorNT(server: Server, err: Error) {
 function _emitListeningNT(server: Server) {
   // Ensure handle hasn't closed
   if (server._handle) {
+    // TODO(duelsik): afaik it differs from node docs, but if it works - let's leave it with
```

Comment:
This is an exact port of the Node code, see https://github.com/nodejs/node/blob/master/lib/net.js#L1414

Reply:
@cmorten I think I did a mistake here, rechecked docs and net.Server doesn't have "socket" pushed alongside listening event. Will fix :)